CMSgov/mint-app
# [NOREF] Archive Model Plan Query

# NOREF

## Changes and Description

- This PR updates the query called in attempting to archive a Model Plan



## How to test this change

1. Create new model plan
2. Click "Remove your Model Plan" 
3. Ensure model plan is removed and no errors
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
